docs/DialogActivityExport_CombinedMode.md

```
@@ -0,0 +1,178 @@
+"uuid2",...  -- From Issue #1716
+...
+"uuid3",...  -- From Issue #1951
+"uuid4",...  -- From Issue #1951
+...
+```
+
+## Progress Reporting
+
+The progress bar shows combined progress:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)